language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -238,6 +238,8 @@ test('create + fetch & exercise', async () => {
     tuple: {_1: '12', _2: 'mmm'},
```

Comment:
This doesn’t seem to actually test the difference anywhere. I admittedly don’t understand typescript but afaict one of the main benefits of the new approach is that I can just use `"Red"` instead of `Color.Red`. We should add some tests for that!

Reply:
Ok, done.